addon/components/nypr-accounts/membership-card.js

```diff
@@ -1,6 +1,33 @@
 import Ember from "ember";
 import layout from "../../templates/components/nypr-accounts/membership-card";
+import computed from "ember-computed";
+import moment from "moment";
+import service from "ember-service/inject";
 
 export default Ember.Component.extend({
   layout,
+  session: service(),
+  previousYear: computed(function() {
+    return moment()
+      .subtract(1, "year")
+      .year();
+  }),
+  pledgesFromLastYear: computed.filter("pledges", function(pledge) {
+    let previousYear = moment()
+      .subtract(1, "year")
+      .year();
```

Comment:
Is there a reason you aren't using the above computed `previousYear` property here?

Reply:
Oops. Just carelessness.